composer.json

```diff
@@ -14,30 +14,30 @@
     "ext-intl": "*",
     "ext-zip": "*",
     "geoip2/geoip2": "~2.4.2",
-    "doctrine/doctrine-bundle": "1.5.2",
+    "doctrine/doctrine-bundle": "~1.5.2",
     "mrclay/minify": "~2.3.0",
     "matthiasmullie/minify": "~1.3.0",
-    "jakeasmith/http_build_url": "^1",
+    "jakeasmith/http_build_url": "~1.0.0",
     "mobiledetect/mobiledetectlib": "~2.8.22",
-    "symfony/swiftmailer-bundle": "2.3.11",
+    "symfony/swiftmailer-bundle": "~2.3.11",
     "pear/archive_tar": "~1.4.2",
     "swiftmailer/swiftmailer": "~5.4.3",
-    "symfony/monolog-bundle": "2.8.2",
-    "sensio/distribution-bundle": "4.0.6",
-    "sensio/framework-extra-bundle": "3.0.10",
+    "symfony/monolog-bundle": "~2.8.2",
+    "sensio/distribution-bundle": "~4.0.6",
+    "sensio/framework-extra-bundle": "~3.0.10",
     "prestashop/translationtools-bundle" : "dev-master",
-    "tecnickcom/tcpdf": "6.2.12",
-    "composer/installers": "1.0.21",
-    "icanboogie/cldr": "1.3.9",
-    "incenteev/composer-parameter-handler": "~2.0",
-    "curl/curl": "1.2.1",
-    "ircmaxell/password-compat": "1.0.4",
+    "tecnickcom/tcpdf": "~6.2.12",
+    "composer/installers": "~1.0.21",
+    "icanboogie/cldr": "~1.3.9",
+    "incenteev/composer-parameter-handler": "~2.0.0",
+    "curl/curl": "~1.2.1",
+    "ircmaxell/password-compat": "~1.0.4",
     "ircmaxell/random-lib": "*",
-    "shudrum/array-finder": "1.1.0",
-    "phpoffice/phpexcel": "1.8",
-    "guzzlehttp/guzzle": "~5.0",
-    "csa/guzzle-bundle": "~1.3",
-    "paragonie/random_compat": "^1.4|^2.0",
+    "shudrum/array-finder": "~1.1.0",
+    "phpoffice/phpexcel": "~1.8.0",
+    "guzzlehttp/guzzle": "~5.0.0",
+    "csa/guzzle-bundle": "~1.3.0",
+    "paragonie/random_compat": "~1.4.0",
```

Comment:
Hello,

I'm currently testing this PR which can solve an issue.
Unfortunately, this change break some relations:
```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - defuse/php-encryption 2.0.3 requires paragonie/random_compat ~2.0 -> satisfiable by paragonie/random_compat[v2.0.0, v2.0.1, v2.0.2, v2.0.3, v2.0.4] but these conflict with your requirements or minimum-stability.
    - defuse/php-encryption 2.0.1 requires paragonie/random_compat ~2.0 -> satisfiable by paragonie/random_compat[v2.0.0, v2.0.1, v2.0.2, v2.0.3, v2.0.4] but these conflict with your requirements or minimum-stability.
    - defuse/php-encryption 2.0.3 requires paragonie/random_compat ~2.0 -> satisfiable by paragonie/random_compat[v2.0.0, v2.0.1, v2.0.2, v2.0.3, v2.0.4] but these conflict with your requirements or minimum-stability.
    - Installation request for defuse/php-encryption ~2.0.1 -> satisfiable by defuse/php-encryption[2.0.1, 2.0.3].
```